lang/talon/talon-code_common_function.talon-list

```diff
@@ -0,0 +1,7 @@
+list: user.code_common_function
+tag: user.talon
+-
+insert: insert
+key: key
+print: print
+repeat: repeat
```

Comment:
Talon list actually supports a single item that will be interpreted as both key and value
`insert: insert` => `insert` are equivalent

Reply:
Do we want to standardize on this approach?